public/docs/ts/latest/cookbook/a11y.jade

```diff
@@ -0,0 +1,1087 @@
+ 
+:marked
+  Click on these violations to drill down further and `aXe` will provide you with useful information on the error, 
+  how to find it in your `HTML` and guidelines to help you fix it:
+  
+figure.image-display
+  img(src="/resources/images/cookbooks/a11y/AXE-Details-Fails.png" alt="aXe a11y violation detail")  
+
+:marked
+  You can rerun aXe and any point during your development and it will show you when there are no `a11y` 
+  violations left:
+  
+figure.image-display
+  img(src="/resources/images/cookbooks/a11y/AXE-Overall-Pass.png" alt="aXe not a11y violations found")
+
+:marked
+  The message in this screenshot is an important one and again stresses that no one automated tool will ensure
+  full `a11y` compliance. And this brings us to another extremely important aspect of `a11y` testing.
+  
+:marked
+  ### The screen reader  
+  
+  When creating web pages it would be preposterous to never open it in a browser before release day. 
+  
+  The same is true for screen readers. The tooling already described should be used to speed up our development
+  cycle, but the only way to truly experience our web pages in a screen reader is by using one.
+  
+  However, using a screen reader for the first time can be a daunting task so here we look at three of the most
+  used screen reader / browser combinations and how to get started with them. You do not have to be an expert in
+  screen reader usage to test your websites for `a11y`!
+  
+.l-sub-section
+  :marked
+    You will use the skills you gained in the `keyboard navigation` section when using a screen reader. First get 
+    comfortable with keyboard-only navigation before adding a screen reader to it. This will help reduce the 
+    confusion that first time users of screen readers usually get confronted by.
+
+.l-sub-section
+  :marked
+    You will not get the same experience in every `screen reader / browser combination`, so do not spend too much time
+    trying to fix *"errors"* in combinations not mentioned here. Often you will be trying to fix a technology 
+    integration issue and not a problem on your page. Focus on the three combinations
+    below and you will ensure that the bulk of users out there can use your websites. 
+    
+:marked
+  #### NVDA (Open source screen reader) in Firefox
+  
+  [NonVisual Desktop Access](http://www.nvaccess.org/), better known as `NVDA` is an open source screen reader for
+  `Microsoft Windows`. For best results, use this screen reader with `Mozilla Firefox`.
+  
+  Both `WebAIM` ([Using NVDA to Evaluate Web Accessibility](http://webaim.org/articles/nvda/)) as well as 
+  `The A11y Project` ([Getting Started with NVDA](http://a11yproject.com/posts/getting-started-with-nvda)) have 
+  excellent introductory guides for getting started with `NVDA`. We recommend that you read either one or both of them. 
+  
+:marked
+  #### JAWS (Most used screen reader) in IE 10+
```

Comment:
Changed this to IE 10/11 in the next commit. Even though there are good things in the pipeline for Edge, there are too many warnings out there to not use Edge. Including the **Freedom Scientific** site itself. So let's limit to these two for now. The docs will evolve with the field as time passes.